### ISSUE TITLE
[Accessibility][Narrator] - Narrator is announcing the status as 'Selected' at every list item of 'Format/Styles/Font name/Font Size'. dropdowns

### ISSUE THREAD

**Issue by sryarama** [ckeditor/ckeditor4#4544]
## Type of report

Bug

## Provide detailed reproduction steps (if any)

1. Launch Chromium EDGE and navigate to CKEDITOR demo url https://ckeditor.com/ckeditor-4/demo/
2. Press ALT + F10 to navigate to toolbar.
3. Expand format dropdown and select Heading 3.
4. Enter some text
5. Start Narrator screen reader.
6. Now press ALT+F10 again and navigate to format dropdown.
7. Observe the screenreader announcement for Heading 1 (Non selected only focused) and observe the announcement for Heading 3 (selected in format)

### Expected result

Screen reader should distinguish between the selected item and focused item. 

### Actual result

Screen reader users may not know which is the selected list item if Narrator announces selected for all list items.

## Other details

* Browser: Chromium EDGE, Narrator Screen reader
* OS: Windows 10
* CKEditor version: 4
* Installed CKEditor plugins: Styles, Font Name, Font Size

**Comment by sculpt0r** [comment]
Hi @sryarama ,

I tested it on Windows 10 and must say that I can't reproduce it. For me, it sounds like valid behavior:

*   Each item on the paragraph format dropdown list is announced by its name and position on the list
*   Only when I move focus to the currently selected item - narrator announce `selected` for it.   
      
    Could you provide an exact version of CKEditor and Edge version?

**Comment by sryarama** [comment]
Hi @sculpt0r,

For me narrator is announcing selected for all the items in the dropdown. 

CKEDITOR version - As I said, I am able to repro it on ckeditor demo page which has version 4.16.0
Edge version - Just to confirm, I am using Chromium EDGE with version "88.0.705.81 (Official build) (64-bit)"

If you are still not able to repro, I can share you a recording Monday.

**Comment by sryarama** [comment]
@sculpt0r - Attached zip has video of issue repro.
[CK Editor repro.zip](https://github.com/ckeditor/ckeditor4/files/6063226/CK.Editor.repro.zip)

**Comment by sculpt0r** [comment]
Hi @sryarama,
Actually, I have Chromium EDGE "89.0.774.45" and I'm still unable to reproduce this issue. Sorry for the delay in response, but could you place this video as for example YT video? As You know - it's rather a bad practice to open zip files from an unknown source.

**Comment by sryarama** [comment]
It is not allowing me to upload files with size greater than 10MB :(

**Comment by sculpt0r** [comment]
@Dumluregn - could you try this one with your PC?

**Comment by Dumluregn** [comment]
Hello @sryarama, I've checked the video you had sent and this is an intended behaviour. Currently there is no possibility to change focused option without selecting it, so reading it by Narrator as `selected` one is expected. It would be a valid issue if keyboard navigation only changed focus and not selection, but it isn't possible now, so I'm closing it.

**Comment by meghanking** [comment]
Thanks @Dumluregn for this feedback. We brought this feedback to our testing team and we also concluded that this is expected Narrator behavior that is not possible to change.
